charts/bulk-scan-payment-processor/values.preview.template.yaml

```diff
@@ -5,6 +5,11 @@ java:
       key: connectionString
   environment:
     PAYMENTS_QUEUE_READ_CONNECTION_STRING: "$(SB_CONN_STRING);EntityPath=payments"
+  keyVaults:
```

Comment:
read it only from `bulk-scan` vault

Reply:
in preview we do not read from  vault? it should be from k8 keystore?